src/main/java/com/puppycrawl/tools/checkstyle/checks/regexp/RegexpOnFilenameCheck.java

```diff
@@ -72,9 +72,7 @@
  * </p>
  * <ul>
  * <li>
- * Property {@code fileExtensions} - Specify the file type extension of files to process. If this is
- * specified, then only files that match these types are examined with the other
- * patterns.
+ * Property {@code fileExtensions} - Specify the file type extension of files to process.
```

Comment:
all other Checks, does not have removed sentence, I think we do not need this sentence.